tests/models/version-test.js

```diff
@@ -231,6 +236,7 @@ module('Model | Version', function (hooks) {
       this.db.version.create({ crate, num: '0.4.3', yanked: true });
       crateRecord = await this.store.findRecord('crate', crate.name, { reload: true });
       versions = (await crateRecord.loadVersionsTask.perform({ reload: true })).slice();
+      crateRecord.release_tracks = calculateReleaseTracks(versions);
```

Comment:
This change was made to pass the tests, but it doesn't reflect what we do in practice (for versions page). Since the new release wouldn't be loaded without a page reload, as already mentioned in the description, I'm not sure what the best approach is in this case.